plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -88,19 +100,69 @@ private void addManageLibrariesWidgets(Composite container) {
     apiGroup.setText(Messages.AppEngineStandardWizardPage_librariesGroupLabel);
     GridDataFactory.fillDefaults().span(2, 1).applyTo(apiGroup);
 
-    for (Library library : getLibraries()) {
+    List<Library> libraries = getLibraries();
+    for (Library library : libraries) {
       Button libraryButton = new Button(apiGroup, SWT.CHECK);
       libraryButton.setText(library.getId());
       libraryButton.setData(library);
       libraryButtons.add(libraryButton);
     }
 
+    addDatabindingForDependencies(libraries);
+
     GridLayoutFactory.fillDefaults().applyTo(apiGroup);
   }
 
+  private void addDatabindingForDependencies(List<Library> libraries) {
+    bindingContext = new DataBindingContext();
+    for (Button libraryButton : libraryButtons) {
+      if (!((Library) libraryButton.getData()).getLibraryDependencies().isEmpty()) {
+        addDatabindingForDependencies(libraryButton);
+      }
+    }
+  }
+
+  private void addDatabindingForDependencies(Button libraryButton) {
+    for (String libraryId : ((Library) libraryButton.getData()).getLibraryDependencies()) {
+      Button dependencyButton = getButtonForLibraryId(libraryId);
+      if (dependencyButton != null) {
+        ISWTObservableValue libraryButtonSelection = WidgetProperties.selection().observe(libraryButton);
+        IObservableValue dependencyButtonSelection =
+            PojoProperties.value(Button.class, "selection").observe(getDisplayRealm(), dependencyButton);
+        IObservableValue dependencyButtonEnablement =
+            PojoProperties.value(Button.class, "enabled").observe(getDisplayRealm(), dependencyButton);
+
+        WritableValue intermediate = new WritableValue(false, Boolean.class);
+        bindingContext.bindValue(libraryButtonSelection, intermediate);
+        bindingContext.bindValue(dependencyButtonSelection, intermediate);
+        bindingContext.bindValue(dependencyButtonEnablement, intermediate,
+                                 new UpdateValueStrategy().setConverter(BooleanConverter.negate()),
+                                 new UpdateValueStrategy().setConverter(BooleanConverter.negate()));
+      }
+    }
+  }
+
+  private Realm getDisplayRealm() {
+    return DisplayRealm.getRealm(getControl().getDisplay());
+  }
+
+  private Button getButtonForLibraryId(String libraryId) {
+    for (Button button : libraryButtons) {
+      if (((Library) button.getData()).getId().equals(libraryId)) {
+        return button;
+      }
+    }
+    return null;
+  }
+
   // mock method until the libraries are defined via an extension point
```

Comment:
perhaps should be a todo comment? also, is this really a mock? It seems functional.